src/components/GroupedTaskStatusBadge/index.tsx

```diff
@@ -0,0 +1,89 @@
+import { css } from "@emotion/react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Body } from "@leafygreen-ui/typography";
+import { taskStatusToCopy } from "constants/task";
+import { TaskStatus } from "types/task";
+
+interface Props {
+  status: TaskStatus;
+  count: number;
+}
+export const GroupedTaskStatusBadge: React.FC<Props> = ({ count, status }) => (
+  <BadgeContainer css={statusToColorVariant[status]}>
+    <Number css={statusToColorVariant[status]}>{count}</Number>
+    <Status css={statusToColorVariant[status]}>
+      {status === TaskStatus.Succeeded || status === TaskStatus.SetupFailed
+        ? [
```

Comment:
we should remove the bracket since so this line evaluates to `string` instaed of `string[]`